### PR TITLE
Rewrite verifiers

### PR DESCRIPTION
**Pull request by MilanPesa**
copied/rewrote the updates from the bug checker branch regarding the verifiers.

### CODE REVIEW

File: verifiers/optimization_verifier.py [hunk 1]
```diff
@@ -39,90 +25,80 @@ def run(self,solver: str, mutations_per_model: int, model_file: str, exclude_dic
...
-                cons = toplevel_list(cons)
-                objective = model.objective_
-                mininimize = model.objective_is_min
-                model = cp.Model(cons)
-                mutators = [copy.deepcopy(cons)] #keep track of list of cons alternated with mutators that transformed it into the next list of cons.
-                if mininimize:
-                    model.minimize(objective)
+                self.cons = toplevel_list(self.cons)
+                self.objective = model.objective_
+                self.mininimize = model.objective_is_min
```
Comment (Wout4) [inline verifiers/optimization_verifier.py]: Typo from old code but maybe it's time to fix it :)

File: verifiers/verifier.py [hunk 2]
```diff
@@ -1,26 +1,183 @@
...
+                            exception=e,
+                            function=function,
+                            argument=argument,
+                            stacktrace=traceback.format_exc(),
+                            mutators=self.mutators,
+                            constraints=self.cons,
+                            )
+            return None
+
+    def initilize_run(self) -> None:
```
Comment (Wout4) [inline verifiers/verifier.py]: typo: initialize

Comment (Wout4) [reply verifiers/verifier.py]: (it's used consistently though)

**Review by Wout4 — COMMENTED** [review]
not many comments, but if I try to run on my machine the verifier immediately stops with this output:

Using solver 'ortools' with models in 'models' and writing to 'output'.

Will use 7 parallel executions, starting...

interrupting...

Executed tests for 0 minutes
Quiting fuzz tests 

Succesfully executed 0 tests, 0 tests failed


I am not doing a keyboard interrupt though, and changing the params doesn't seem to help

**Comment by MilanPesa** [comment]
hmm strange that it is not running on your computer. There probably occurred an exception that i didn't handle decently.  @Wout4 can you run it again with the updated code, also which OS are you running it on?

**Comment by Wout4** [comment]
it's on windows, now i get this error:
An unexcpected error occured error:
Cannot close a process while it is still running. You should first call join() or terminate(). 
stacktrace:
Traceback (most recent call last):
  File "C:\Users\wout\PycharmProjects\cpmpy2\cpmpy\fuzztest\fuzz_test.py", line 79, in <module>
    process.close()
  File "C:\Users\wout\.conda\envs\cpmpy2_dev\lib\multiprocessing\process.py", line 181, in close
    raise ValueError("Cannot close a process while it is still running. "
ValueError: Cannot close a process while it is still running. You should first call join() or terminate().


Executed tests for 0 minutes
Quiting fuzz tests 

Succesfully executed 0 tests, 0 tests failed

**Comment by MilanPesa** [comment]
@Wout4 it should be fixed now.

**Pull request merged**